core/domain/exp_services_test.py

```diff
@@ -5648,3 +5732,3136 @@ def test_when_few_state_stats_models_are_missing_for_old_exps(self):
+                            'data_format': 'html'
+                        }
+                    },
+                    'hint_1': {},
+                    'ca_placeholder_0': {},
+                    'default_outcome': {},
+                    'solution': {}
+                }
+            },
+            'new_value': {
+                'translations_mapping': {
+                    'content': {
+                        'fr': {
+                            'translation': '<p>Translation Content 1.</p>',
+                            'needs_update': False,
+                            'data_format': 'html'
+                        }
+                    },
+                    'hint_1': {},
+                    'ca_placeholder_0': {
+                        'fr': {
+                            'translation': 'Translation Placeholder.',
+                            'needs_update': False,
+                            'data_format': 'unicode'
+                        }
+                    },
+                    'default_outcome': {},
+                    'solution': {}
+                }
+            },
+            'state_name': 'Introduction'
+        })]
+
+        exp_services.update_exploration(
+            self.owner_id, self.EXP_0_ID, change_list_2,
+            'Added some translations.')
+
+        # Adding translations again to the same first state
+        # to check if they can be applied. They will not
+        # be mergeable as the changes are in the same property
+        # i.e. written_translations.
+        change_list_3 = [exp_domain.ExplorationChange({
+            'cmd': 'edit_state_property',
+            'old_value': {
+                'translations_mapping': {
+                    'hint_1': {},
+                    'content': {},
+                    'ca_placeholder_0': {},
+                    'solution': {},
+                    'default_outcome': {}
+                }
+            },
+            'new_value': {
+                'translations_mapping': {
+                    'hint_1': {
+                        'fr': {
+                            'translation': '<p>Translation Hint 2.</p>',
+                            'needs_update': False,
+                            'data_format': 'html'
+                        }
+                    },
+                    'content': {},
+                    'ca_placeholder_0': {},
+                    'solution': {},
+                    'default_outcome': {}
+                }
+            },
+            'property_name': 'written_translations',
+            'state_name': 'Introduction'
+        })]
+
+        are_changes_mergeable = exp_services.are_changes_mergeable(
+            self.EXP_0_ID, 2, change_list_3)
+        self.assertEqual(are_changes_mergeable, False)
+
+        # Adding translations to the second state to check
+        # if they can be applied. They can be mergead as
+        # the changes are in the differents states.
```

Comment:
```suggestion
        # the changes are in different states.
```

Reply:
Done.

Reply:
Not done? Also, please do not resolve comments. It makes it harder for the reviewers to look at.

Reply:
Now changed it.